### DISCUSSION TITLE
about using yalmip in Hardware-in-the-Loop(Hil)

### DISCUSSION THREAD

**Discussion by JasonLiu555** [yalmip/YALMIP#1040] [General discussion]
Can yalmip support hardware in the loop? I wrote the control strategy in matlab/simulink with using yalmip, can it be compiled into C or C++?

**Comment by johanlofberg** [comment]
It depends on the solver (does it allowed being compiled) and then you have to realize that the compiled code will essentially be a stand-alone MATLAB container (at least that is what happened when I did something related some 10 years ago, leading to a 175MB binary). And then this https://yalmip.github.io/faq/compilesolver/

**Reply by JasonLiu555** [reply]
thanks，for example, like sedumi，I need sedumi.m, ada_pcg.m,install_sedumi.m,callsedumi.m,anything else?

**Reply by johanlofberg** [reply]
Those are the ones listed. Haven't done this is in a decade though so no idea ho todays compiler work
